.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
```

Comment:
Please remove this file `git rm -r --cached .vscode`

Reply:
Similar command to remove the database. 
`git rm -r --cached lmnop.sqlite`

Reply:
pulled the repo and fixed the issue.